test/integration/supported/client-pool-tests.js

```diff
@@ -4,17 +4,17 @@ const { assert } = require("chai");
 const dns = require("dns");
 const util = require("util");
 
-const helper = require("../../test-helper");
-const Client = require("../../../lib/client");
-const clientOptions = require("../../../lib/client-options");
-const utils = require("../../../lib/utils");
-const errors = require("../../../lib/errors");
-const types = require("../../../lib/types");
-const policies = require("../../../lib/policies");
+const helper = require("../../test-helper.js");
+const Client = require("../../../lib/client.js");
+const clientOptions = require("../../../lib/client-options.js");
+const utils = require("../../../lib/utils.js");
+const errors = require("../../../lib/errors.js");
+const types = require("../../../lib/types/index.js");
+const policies = require("../../../lib/policies/index.js");
```

Comment:
:question: Why now do we need to add the `.js` suffix and previously we didn't have to?